src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -141,6 +143,9 @@ def get(self, request: Request, organization) -> Response:
             cursor_cls=StringCursor if case_insensitive else Cursor,
             case_insensitive=case_insensitive,
         )
+        response["X-Sentry-Issue-Rule-Hits"] = issue_rules_count
+        response["X-Sentry-Alert-Rule-Hits"] = alert_rules_count
```

Comment:
@Neo-Zhixing I think this change makes sense to me. If we get the OK from someone on the Workflow team let's also add some tests.